docs/content/deployment/dagster-instance.mdx

```diff
@@ -731,18 +731,18 @@ compute_logs:
 
 ### Local Artifact Storage
 
-The local artifact storage is used to configure storage for any artifacts that require a local disk, such as schedules, or when using the filesystem system storage to manage files and intermediates.
+The local artifact storage is used to configure storage for any artifacts that require a local disk, such as schedules, or when using the filesystem IO manager to manage files and intermediates.
```

Comment:
I think you can remove the "files and" - that looks like a reference to when the system storage supplied the file manager in addition to the IO manager.